app/services/paypal_service/logger.rb

```diff
@@ -2,29 +2,28 @@ module PaypalService
   class Logger
 
     def info(msg)
-      #TODO: Fix logging totally
-      Rails.logger.warn "[Paypal] #{msg}"
+      Rails.logger.info ({ paypal: msg }.to_json)
```

Comment:
Is there a reason why this is a bit inconsistent compared to other log levels? Shouldn't it be `Rails.logger.warn ({ paypal: { type: :info, message: msg } }.to_json)`